recipe/meta.yaml

```
@@ -1,5 +1,5 @@
 {% set name = "openmm-torch" %}
-{% set version = "0.8" %}
+{% set version = "1.0beta" %}
```

```suggestion
{% set version = "1.0b" %}
```

I think `conda` prefers single letter suffixes

---

I don't think so?  `rc#` is the [recommended suffix for release candidates](https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-version-sorting), so I'm pretty sure it's fine with multiple letters.